config/apPipe.py

```diff
@@ -32,8 +32,8 @@
     "y": "y",
     "VR": "g"}
 
-# Templates are deepCoaddPsfMatched
+# Templates are deepCoadds
```

Comment:
Just to make sure: these are not supposed to be CompareWarps?

Reply:
Well, they are constructed with the CompareWarm algorithm, but the butler type is deep.

Reply:
Can you change the comment to explain the distinction? Looking at this file one would think they're plain old direct coadds.